src/main/java/me/greenleaf/Legolas.java

```diff
@@ -1,24 +1,45 @@
 package me.greenleaf;
 
+import java.util.ArrayList;
+import java.util.Random;
+
 public class Legolas {
-    private SASS_LEVELS sass;
+    private SASS_LEVEL sass;
+    private static Legolas legolasInstance;
 
-    public static void main(String[] args) {
-        Legolas.getInstance().setSass(SASS_LEVELS.LOW);
+    private Legolas() {
+        sass = SASS_LEVEL.HIGH;
     }
 
     public static Legolas getInstance() {
-        return new Legolas();
+        if (legolasInstance == null) {
+            legolasInstance = new Legolas();
+        }
```

Comment:
This would be better done with a static class instance holder. See https://stackoverflow.com/questions/15019306/regarding-static-holder-singleton-pattern.